CONTRIBUTION.md

```diff
@@ -0,0 +1,25 @@
+## Building the Docker Image
+
+**This step is only required, if you work at the build environment!**
+
+Build the Docker image inside the project root directory.
+
+```shell
+docker build -f docker/Dockerfile -t chrgernoe/yacguide .
```

Comment:
Isn't that the wrong owner (chrgernoe)?

Reply:
That's my username/namespace at hub.docker.com. I will change this later by creating a organization YacTeam at hub.docker.com, too.

Reply:
Created organization yacgroup at hub.docker.com which now provides the Docker image.